shared/actions/chat/shared.js

```diff
@@ -81,8 +81,7 @@ function* startNewConversation(
     console.warn("Shouldn't happen in practice")
     return [null, null]
   }
-
-  const membersType = getenv('KEYBASE_CHAT_MEMBER_TYPE', 'kbfs') === 'impteam'
+  const membersType = flags.admin
```

Comment:
you should really make a feature flag for the feature. you can say you want it on for admins. we don't refer to admin anywhere directly